### DISCUSSION TITLE
Can I get item personalization data from an order?

### DISCUSSION THREAD

**Discussion by mackemode** [etsy/open-api#1031] [Q&A]
My listings accept personalizations with a text field. Is there any way to get the content of this personalization on each order through the API? I've explored the getShopReceipt end point but I'm not seeing anything about this personalization field. This is different from the gift_message, as the gift_message is an order level field and I"m looking for an item level field of the personalization notes for each item on the order.

![image](https://github.com/etsy/open-api/assets/27908805/8a382bd1-69a9-4100-a348-2accbd917249)

**Comment by Gareth064** [comment]
Ho @mackemode 

The personalisation appears as a variation in the transaction.

Receipt > Transactions > Variations.
